strongbox-web-core/pom.xml

```
@@ -51,6 +51,34 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <target>
+                        1.9
+                    </target>
+                    <compilerArgs>
+                        <arg>
+                            --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED 
+                        </arg>
+                        <arg>
+                            --add-exports=java.base/jdk.internal.ref=ALL-UNNAMED
+                        </arg>
+                        <arg>
+                            --add-exports=java.base/sun.nio.ch=ALL-UNNAMED 
+                        </arg>
+                        <arg>
+                            --add-exports=java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED 
+                        </arg>
+                        <arg>
+                            --add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED 
+                        </arg>
+                        <arg>
+                            --add-exports=java.rmi/sun.rmi.server=ALL-UNNAMED 
+                        </arg>
+                        <arg>
+                            --add-exports=java.sql/java.sql=ALL-UNNAMED 
+                        </arg>
```

Cosmetic: These parameters could be one-liners:
```suggestion
                    <target>1.9</target>
                    <compilerArgs>
                        <arg>--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.base/jdk.internal.ref=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.base/sun.nio.ch=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.rmi/sun.rmi.server=ALL-UNNAMED</arg>
                        <arg>--add-exports=java.sql/java.sql=ALL-UNNAMED</arg>
```